src/motion/cursor.ts

```diff
@@ -0,0 +1,33 @@
+import * as vscode from 'vscode';
+
+export default class Cursor {
```

Comment:
A nicety introduced in ES6 to make it easier to import modules --  http://www.2ality.com/2014/09/es6-modules-final.html